i18n/currency-info.php

```diff
@@ -0,0 +1,952 @@
+<?php
+/**
+ * Currency formatting information
+ *
+ * @package WooCommerce\i18n
+ * @version 5.5.0
```

Comment:
Can you please change the version to 5.6.0

Reply:
Done.